src/plugins/vis_type_timelion/public/timelion_vis_type.tsx

```diff
@@ -48,11 +48,6 @@ export function getTimelionVisDefinition(dependencies: TimelionVisDependencies)
         expression: '.es(*)',
         interval: 'auto',
       },
-      component: (props: TimelionVisComponentProp) => (
-        <KibanaContextProvider services={{ ...dependencies }}>
-          <TimelionVisComponent {...props} />
-        </KibanaContextProvider>
-      ),
     },
     editorConfig: {
       optionsTemplate: (props: TimelionOptionsProps) => (
```

Comment:
I know this is technically out of scope for this PR, but I think we could lazy-load `TimelionOptions` as well to shave off a few additional KB from the initial bundle size - wdyt?

Reply:
Makes sense! Especially since it loads monaco 🤔

Reply:
Done!
![image](https://user-images.githubusercontent.com/31325372/94790943-ab368780-03df-11eb-96e2-e41cc1ba6db0.png)

Tested against the default editor PR, works well